test_scripts/Defects/8_1/3838/3838_5_SDL_does_not_send_UI.ClosePopUp_BOTH_mode_in_case_VR.PerformInteraction_without_choiceID.lua

```diff
@@ -0,0 +1,126 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/3838
+---------------------------------------------------------------------------------------------------
+-- Description: Check that SDL does not send UI.ClosePopUp to HMI in case:
+-- - mobile App sends PerformInteraction request with BOTH interaction mode to SDL
+-- - HMI responds with 'SUCCESS' to VR.PerformInteraction request without choiceID
+--
+-- Pre-conditions:
+-- 1. SDL, HMI, Mobile session are started
+-- 2. App is registered and activated
+-- 3. CreateInteractionChoiceSet is added
+--
+-- Steps:
+-- 1. App sends PerformInteraction request with BOTH interaction mode to SDL
+-- 2. HMI responds with 'SUCCESS' to VR.PerformInteraction request without choiceID
+-- SDL does:
+-- - not send UI.ClosePopUp request to HMI
+-- - send PerformInteraction response (resultCode: SUCCESS, success:true) to mobile App
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require("test_scripts/Defects/8_1/3838/common")
+
+--[[ Local Variables ]]
+local initialPromptValue = common.getPromptValue(" Make your choice ")
+
+local helpPromptValue = common.getPromptValue(" Help Prompt ")
+
+local timeoutPromptValue = common.getPromptValue(" Time out ")
+
+local vrHelpvalue = {
+  {
+    text = " New VRHelp ",
+    position = 1
+  }
+}
+
+local requestParams = {
+  initialText = "StartPerformInteraction",
+  initialPrompt = initialPromptValue,
+  interactionMode = "BOTH",
+  interactionChoiceSetIDList = { 100 },
+  helpPrompt = helpPromptValue,
+  timeoutPrompt = timeoutPromptValue,
+  timeout = 5000,
+  vrHelp = vrHelpvalue,
+  interactionLayout = "ICON_ONLY"
+}
+
+--[[ Local Functions ]]
+local function setExChoiceSet(pChoiceIDValues)
+  local exChoiceSet = { }
+  for i = 1, #pChoiceIDValues do
+    exChoiceSet[i] = {
+      choiceID = pChoiceIDValues[i],
+      menuName = "Choice" .. pChoiceIDValues[i]
+    }
+  end
+  return exChoiceSet
+end
+
+local function PI_ViaBOTH(pParams)
+  local cid = common.getMobileSession():SendRPC("PerformInteraction",pParams)
+  common.getHMIConnection():ExpectRequest("UI.ClosePopUp", { methodName = "UI.PerformInteraction" }):Times(0)
+  common.getHMIConnection():ExpectRequest("VR.PerformInteraction", {
+      helpPrompt = pParams.helpPrompt,
+      initialPrompt = pParams.initialPrompt,
+      timeout = pParams.timeout,
+      timeoutPrompt = pParams.timeoutPrompt
+    })
+  :Do(function(_,data)
+      common.getHMIConnection():SendNotification("VR.Started")
+      common.getHMIConnection():SendNotification("TTS.Started")
+      common.sendOnSystemContext("VRSESSION")
+      local function firstSpeakTimeOut()
+        common.getHMIConnection():SendNotification("TTS.Stopped")
+        common.getHMIConnection():SendNotification("TTS.Started")
```

Comment:
What's the reason behind restarting TTS here?

Reply:
@jacobkeeler There is no reason to restart notifications of TTS.Started  to speak the 'timeoutPrompt ' Done in [c82d72](https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2620/commits/fc82d72c5697bc44064f72c26ce303b8fdf563cc)